src/main/webapp/js/actions/index.js

```diff
@@ -125,8 +125,10 @@ export const updateEntity = (entityType, entity) => {
   };
 };
 
-export const createEntity = (entityType, entity) => {
+export const createEntity = (entityType, entity, callback) => {
   const request = apiClient.post(`${BASE_URL}/${entityType}`, entity);
+  request.then(() => callback());
```

Comment:
I think redux-thunk should be used here.

Reply:
there is no need for redux-thunk here, it's just a single action